sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/trees/TreeNode.scala

```diff
@@ -886,7 +886,8 @@ abstract class TreeNode[BaseType <: TreeNode[BaseType]] extends Product with Tre
 
   private def stringArgsForCatalogTable(table: CatalogTable): Seq[Any] = {
     table.storage.serde match {
-      case Some(serde) => table.identifier :: serde :: Nil
+      case Some(serde) if SQLConf.get.excludeSerdeOnCatalogTableFromExplain =>
```

Comment:
```suggestion
      case Some(serde) if !SQLConf.get.excludeSerdeOnCatalogTableFromExplain =>
```

Reply:
Is it correct?

Reply:
lol I changed the code twice and I missed this. nice catch!